generics/helpers/files_helper.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
```

Comment:
@MallanagoudaB we already have constants/common.js file. move these constants there, check for redundancy also